docs/python-support.md

```diff
@@ -0,0 +1,39 @@
+# Python Support in Djinni
+
+Python support in Djinni is experimental, but ready to try out.  It can generate code for bridging C++ with Python 3.
+
+For more information, you can check out our talk from CppCon 2015.  [Slides](https://bit.ly/djinnitalk2) and [video](https://bit.ly/djinnivideo2) are avilable online.
+
+## Generated Code
+
+When bridging to Python, Djinni generates 4 types of output:
+
+* `python` Generated Python classes and proxies for interacting with C++ via [CFFI](https://cffi.readthedocs.org/).
+* `cffi` Python code run at build time to create a Python extension out of the C++ code.
+* `cwrapper` A C interface implemented in C++ to allowing Python to interact with C++ classes.
+* `cpp` The same C++ classes generated for all other Djinni languages.
```

Comment:
I'd prefer to see documentation related to the generation of gluecode in the generator docs. I think it would best fit [here](https://djinni.xlcpp.dev/djinni-generator/generated-code-usage/).

As a general rule of thumb I think it makes sense to put documentation that is closely related to a component into the components repository, while general information about python support could be put it in the general documentation thats not component-related: ["Getting Started"](https://djinni.xlcpp.dev/overview/)

The Markdown files for the general documentation is located [here](https://github.com/cross-language-cpp/cross-language-cpp.github.io/tree/main/docs).